frontend/src/cards/MapCard.tsx

```diff
@@ -48,100 +49,71 @@ function MapCardWithKey(props: MapCardProps) {
   };
 
   // TODO - make sure the legends are all the same
-  // TODO - pull these from the data itself
-  const RACES = props.nonstandardizedRace
-    ? [
-        "Total",
-        "American Indian and Alaska Native",
-        "American Indian and Alaska Native (Non-Hispanic)",
-        "Asian",
-        "Asian (Non-Hispanic)",
-        "Black or African American",
-        "Black or African American (Non-Hispanic)",
-        "Hispanic or Latino",
-        "Native Hawaiian and Pacific Islander",
-        "Native Hawaiian and Pacific Islander (Non-Hispanic)",
-        "Some other race",
-        "Some other race (Non-Hispanic)",
-        "Two or more races",
-        "Two or more races (Non-Hispanic)",
-        "White",
-        "White (Non-Hispanic)",
-      ]
-    : [
-        "American Indian/Alaskan Native, Non-Hispanic",
-        "Asian, Non-Hispanic",
-        "Black, Non-Hispanic",
-        "Hispanic",
-        "Other race, Non-Hispanic",
-        "White, Non-Hispanic",
-      ];
-
-  const [breakdownFilter, setBreakdownFilter] = useState<string>(RACES[0]);
+  const [breakdownFilter, setBreakdownFilter] = useState<string>("");
   const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
 
   const datasetStore = useDatasetStore();
 
-  let queries = [];
-  let initalQuery: MetricQuery;
-
-  if (["age", "all"].includes(props.currentBreakdown)) {
-    const ageQuery = new MetricQuery(
-      props.metricConfig.metricId,
-      Breakdowns.byState().andAge()
-    );
-    queries.push(ageQuery);
-    if (props.currentBreakdown !== "all") {
-      initalQuery = ageQuery;
-    }
-  }
-  if (["sex", "all"].includes(props.currentBreakdown)) {
-    const sexQuery = new MetricQuery(
-      props.metricConfig.metricId,
-      Breakdowns.byState().andGender()
-    );
-    queries.push(sexQuery);
-    if (props.currentBreakdown !== "all") {
-      initalQuery = sexQuery;
+  let queries: Record<string, MetricQuery> = {};
+  ["race_and_ethnicity", "age", "sex"].forEach((possibleBreakdown) => {
+    if (
+      props.currentBreakdown === possibleBreakdown ||
+      props.currentBreakdown === "all"
+    ) {
+      queries[possibleBreakdown] = new MetricQuery(
+        props.metricConfig.metricId,
+        Breakdowns.byState().addBreakdown(
+          possibleBreakdown as BreakdownVar,
```

Comment:
can you avoid the typecast by declaring `const possibleBreakdowns: BreakdownVar[] = ["race_and_ethnicity", "age", "sex"]`?